string label
float32 up_alpha
float32 down_alpha
float32 up_beta
float32 down_beta
